src/zmq/zmqnotificationinterface.cpp

```diff
@@ -118,7 +122,7 @@ void CZMQNotificationInterface::Shutdown()
             LogPrint("zmq", "   Shutdown notifier %s at %s\n", notifier->GetType(), notifier->GetAddress());
             notifier->Shutdown();
         }
-        zmq_ctx_destroy(pcontext);
+        zmq_ctx_term(pcontext);
 
         pcontext = 0;
```

Comment:
better to use `nullptr`.

Reply:
Ok. This line was taken verbatim from the Bitcoin Core upgrade